package.json

```diff
@@ -39,28 +39,28 @@
     "test.watch": "yarn test --watch"
   },
   "dependencies": {
-    "js-yaml": "^3.13",
+    "@stoplight/yaml": "^4.0.2",
     "lodash": "^4.17.15",
-    "mdast-util-to-string": "^1.0.8",
+    "mdast-util-to-string": "^1.1.0",
     "remark-frontmatter": "^1.3",
-    "remark-parse": "^7.0.2",
-    "remark-stringify": "^7.0.4",
-    "tslib": "^1.11.1",
-    "unified": "^8.4.2",
+    "remark-parse": "^8.0.2",
+    "remark-stringify": "^8.0.0",
+    "tslib": "^1.13.0",
+    "unified": "^9.0.0",
     "unist-util-select": "^3.0.1"
   },
   "devDependencies": {
     "@stoplight/scripts": "~8.2.0",
-    "@stoplight/types": "^11.4.0",
-    "@types/jest": "^25.1.3",
-    "@types/js-yaml": "3.12.2",
-    "@types/lodash": "^4.14.149",
-    "@types/unist": "2.0.3",
-    "jest": "^25.1.0",
-    "ts-jest": "^25.2.1",
+    "@stoplight/types": "^11.6.0",
+    "@types/jest": "^25.2.3",
+    "@types/lodash": "^4.14.152",
+    "@types/unist": "^2.0.3",
+    "jest": "^26.0.1",
+    "js-yaml": "^3.13.1",
```

Comment:
Do we need `js-yaml` in here anymore?

Reply:
Nope! Was unused. Removed. Thanks for spotting.